features/admin.remote-repository-configuration.v1/components/remote-fetch-status.tsx

```diff
@@ -58,7 +62,7 @@ export const RemoteFetchStatus: FunctionComponent<RemoteFetchStatusProps> = (
     const [ openRemoteFetchDetails, setOpenRemoteFetchDetails ] = useState<boolean>(false);
     const [ remoteConfig, setRemoteConfig ] = useState<InterfaceRemoteRepoConfig>(undefined);
 
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
Let's import `Dispatch` from `redux` instead of react.

```suggestion
    const dispatch: Dispatch = useDispatch();
```

Reply:
Sure!